hugo/content/blog/2024-05-19-mumble-1-5-634.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Mumble 1.5.634
+author: Hartmnt
+date: 2024-05-19
+categories:
+  - Release
+---
+
+After quite some time, we are proud to present you with the first stable release of the Mumble 1.5.x series. While this release may not be groundbreaking
+in terms of flashy and new features, we implemented a lot of quality-of-life and structural improvements to provide a solid foundation for the
+future development of the project.
+
+You can download the new version from our [**Downloads page**]({{< relref "/downloads" >}}) or the
+[**GitHub release page**](https://github.com/mumble-voip/mumble/releases/tag/v1.5.634) or from within your Windows client or software package
+management system.
+
+<!--more-->
+
+# Mumble 1.5.634
+
+Almost two years after the previous stable release Mumble 1.4.287, a lot of changes have been made to the codebase. With this update, you will surely
+notice some long-standing bugs fixed, as well as some convenient changes that were long overdue. In the time between the major versions, the project has
+exceeded the 10,000 commit mark and now has over 350 contributors. Thanks to all of you who contributed, translated, submitted issues, and tested
+pre-releases! Without you, the Mumble project would not be possible at this scale ♥
+
+While the changelog is quite long, the amount of notable client changes is relatively small. For this release, we focus on the few main changes.
+For the interested reader, we of course also include a full changelog at the bottom of this page as well.
+
+If you are wondering, why this first 1.5.x release is not version 1.5.0, but 1.5.634, please see
+[this old blog post describing our _new versioning scheme_]({{< relref "/blog/2021-12-07-new-versioning-scheme/index.md" >}}) that
+explains the absence of the mentioned version numbers.
+
+
+## Feature spotlight
+
+### Positional audio improvements
+
+There have been some changes to the positional audio system, especially to better fine-tune the configuration. The maximum distance thresholds
+have been significantly increased. The option to set a minimum volume of 0 has been brought back. And some clever psychoacoustic effects, such as 
+[interaural delay](https://en.wikipedia.org/wiki/Interaural_time_difference) as well as improved blooming will enhance the positional audio experience
+even more.
+
+For those users who wish to develop positional audio plugins - or try to debug them - we have added a convenient "Positional Audio Viewer" to the
+developer context menu. This will show you the positional information of your client and the associated metadata in real-time.
+
```

Comment:
Let's mention that the feature was previously provided in the form of a standalone program called MumblePAHelper (Mumble Positional Audio Helper).

There's also a video showcasing the feature in Mumble: https://odysee.com/@Mumble:5/positional-audio-viewer-left4dead:f